src/components/WorkOrder/AddInventoryItemsFields.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react';
+import Select from 'react-select';
+
+import { INVENTORY_ITEMS_CONDITION_OPTIONS, FIELDS } from './formConfig';
+import api from '../../queries/api';
+
+export default class AddInventoryItemsFields extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      itemOptions: [],
+    };
+  }
+
+  componentDidMount() {
+    this.getItemOptions();
+  }
+
+  getItemOptions = () => {
+    api
+      .workOrder()
+      .getInventoryItems()
+      .then(res => {
+        console.log(res);
+        const itemOptions = res.data.records.map(item => {
+          return { label: item.identifier, value: item.id };
+        });
+        this.setState({ itemOptions });
+      });
+  };
+
+  handleConditionChange = e => {
+    let data = {};
+    data[FIELDS.WORK_ORDER_ITEM_CONDITION] = e.target.value;
+    this.props.handleItemPropertyChange(data);
+  };
+
+  handleQuantityChange = e => {
+    let data = {};
+    data[FIELDS.WORK_ORDER_ITEM_QUANTITY] = e.target.value;
+    this.props.handleItemPropertyChange(data);
+  };
+
+  render() {
+    return (
+      <>
+        <div className="form-group">
+          <label htmlFor={FIELDS.WORK_ORDER_INVENTORY_ITEMS}>
+            Inventory Item
+          </label>
+          <Select
+            className="basic-single"
+            classNamePrefix="select"
+            defaultValue={''}
+            isClearable
+            isSearchable
+            name={FIELDS.WORK_ORDER_INVENTORY_ITEMS}
+            options={this.state.itemOptions}
+            onChange={this.props.handleInventoryItemChange.bind(
+              this,
+              FIELDS.WORK_ORDER_INVENTORY_ITEMS
+            )}
+            required // Prevent blank item from adding to DB since Knack does not require these fields
```

Comment:
This might be something we want to flag for @dianamartin to see if she wants to add validation on the Knack app side.

Reply:
Good idea so we can keep form validation consistent.

@dianamartin Do you want validation on the fields below for inventory items? Knack allowed me to add an empty record without POSTing any values for them so I ended up requiring the fields in the React forms. Let me know what you decided, and I'll update the code if needed.

**scene_297 & view_889**
field_513 (item)
field_524 (item quantity)
field_1071 (item condition)

Reply:
Knack form validation for these fields is now addressed in issue #134. Closing this issue and merging.